wordpress-mobile/WordPress-iOS
# [Stats Refresh] Make stat section labels accessible

Fixes #n/a

While working on the Stats detail list view, I realized it needed access to the strings for the stat cards (i.e. card title and column titles). However, these strings were private and in different view models. So I took advantage of the `StatSection` enum, and moved all the strings out of the view models and into `StatSection`. And since it was a considerable refactor, it gets its very own PR.

Now, these strings can be accessed just by knowing the `StatSection`. As an added bonus, this refactor allowed several duplicate strings to be removed. 🎉 

To test:
Visibly, nothing should change. Just verify all the cards have the correct card titles and column titles (where applicable), i.e. these three labels:

<img width="350" alt="strings" src="https://user-images.githubusercontent.com/1816888/53134604-0d0ceb00-3535-11e9-9059-c8858e78cad0.png">
